subgraphs/curve-finance/src/modules/Deposit.ts

```diff
@@ -41,19 +37,19 @@ export function createDepositTransaction(
   if (!depositTransaction) {
     depositTransaction = new DepositTransaction(transactionId);
 
-    depositTransaction.pool = liquidityPool.id;
+    depositTransaction.pool = pool.id;
     depositTransaction.protocol = getOrCreateDexAmmProtocol().id;
 
-    depositTransaction.to = liquidityPool.id;
+    depositTransaction.to = pool.id;
     depositTransaction.from = provider.toHexString();
 
     depositTransaction.hash = transaction.hash.toHexString();
     depositTransaction.logIndex = transaction.index.toI32();
 
-    depositTransaction.inputTokens = inputTokens;
+    depositTransaction.inputTokens = pool._inputTokensOrdered;
```

Comment:
Will this work? `Deposit.inputTokens` is ` [Token]`, so when querying the order will change

Reply:
yeah, you are right. This won't work.